content/en/platform/corda/5.2/deploying-operating/deployment/deploying/_index.md

```diff
@@ -414,108 +411,399 @@ The following configuration is required for the Corda databases:
 * [Cluster Database](#cluster-database)
 * [State Manager Databases](#state-manager-databases)
 
+Corda supports using multiple databases in isolation to spread the load and improve horizontal scalability.
+All databases must be defined only once within the deployment configuration and any dependant component can simply reference the database by `id`. This greatly simplifies maintenance and reduces redundant information.
+
+{{< note >}}
+A database with an `id` set as `default` is always mandatory. Corda uses this database if nothing else is configured.
+{{< /note >}}
+
 #### Cluster Database
 
-The password for PostgreSQL can be specified directly as a Helm override, but this is not recommended.
-Instead, create a Kubernetes secret containing the password with a key of `password`. By default, the installation expects a database named `cordacluster`, but this can be overridden. You can then define the PostgreSQL configuration as follows:
+The following Corda workers require access to the cluster database, and credentials must be specified for each one independently.
+
+* Crypto Workers
+* Database Workers
+* Persistence Workers
+* Token Selection Workers
+* Uniqueness Workers
+
+By default, Corda expects a database named `cordacluster`, but you can override this, as follows:
 
 ```yaml
-db:
-  cluster:
-    host: <POSTGRESQL_HOST>
+databases:
+  - id: "default"
+    host: <POSTGRESQL_HOST1>
+    name: <POSTGRESQL_CLUSTER_DATABASE_NAME>
     port: 5432
-    username:
-      value: <POSTGRESQL_USER>
-    password:
-      valueFrom:
-        secretKeyRef:
-          name: <POSTGRESQL_PASSWORD_SECRET_NAME>
-          key: <POSTGRESQL_PASSWORD_SECRET_KEY>
-    database: <POSTGRESQL_CLUSTER_DB>
+    type: "postgresql"
+```
+
+A single set of credentials can be specified by referencing, under each worker, the following configuration that defines a Kubernetes secret containing the `username` and `password`:
+
+```yaml
+config:
+  username:
+    valueFrom:
+      secretKeyRef:
+        key: <POSTGRESQL_CLUSTER_DATABASE_USERNAME_SECRET_KEY>
+        name: <POSTGRESQL_CLUSTER_DATABASE_USERNAME_SECRET_NAME>
+  password:
+    valueFrom:
+      secretKeyRef:
+        key: <POSTGRESQL_CLUSTER_DATABASE_PASSWORD_SECRET_KEY>
+        name: <POSTGRESQL_CLUSTER_DATABASE_PASSWORD_SECRET_NAME>
+```
```

Comment:
I don't think the Helm chart does support this, does it? You have to provide credentials per-worker.

Reply:
I think this comes back to my original confusion. I think maybe we should have an example showing the reference from under a worker?

Reply:
Fixed.